packages/usa-footer/src/styles/_usa-footer.scss

```diff
@@ -166,11 +153,7 @@ $-chevron-expand-more: map-merge(
   line-height: line-height($theme-footer-font-family, 2);
 
   a {
-    @include u-text("ink", "no-underline");
-
-    &:hover {
-      text-decoration: underline;
-    }
```

Comment:
Additional styles redundant due to primary links having an underline in default states now